### ISSUE TITLE
User/Group IDs - not mapping correctly?

### ISSUE THREAD

**Issue by Juggler00** [jlesage/docker-crashplan-pro#239]
Running this on my home server. I have created a crashplan user with 1004/1005 uid/gid. 
```
cat /etc/passwd
...
crashplan:x:1004:1005::/home/crashplan:/bin/sh
...
```

I currently have the following portion in my docker-compose file:
```
    environment:
      - TZ=America/Toronto
      - USER_ID=1004
      - GROUP_ID=1005
```
However, when I connect to the container and do the following:
`touch /storage/temp`
The file outside of the container is mapped to root instead of the the crash plan user:
`-rw-r--r--  1 root          root           0 May 20 15:17 temp`

Is there anything obvious that I'm doing wrong? Is this a bug?

**Comment by jlesage** [comment]
When you login to the container, you are starting by default a shell as root.

`USER_ID` and `GROUP_ID` variables are not related to this: they are used to set the user under which CrashPlan runs.

**Comment by Juggler00** [comment]
Ah. I was hoping that would explain the problem I'm having; I must be doing something else wrong.

As per above, I have configured the uid/gid of the container to map to user "crashplan" on my server. The files I'm trying to backup are all located in /hdd12tbpool/data. In each of the directories listed below are further subdirectories, all with group set to "crashplan"
```
user@docker:/hdd12tbpool/data$ ll
total 28
drwxr-xr-x  7 root          root  7 May 20 17:32 ./
drwxr-xr-x  4 root          root  4 May  4 18:00 ../
drwxrwx---  4 plex          crashplan  5 May 20 17:41 media/
drwxrwx---  2 homeassistant crashplan  5 May 21 01:11 security/
drwxrws--- 21 user        crash plan 32 May 19 13:21 shared/
```
The problem is that when I log into the crashplan container's webui, click on "Manage Files" and navigate to "storage", I can see the above directories but they are all showing as empty. 

Is there a log that I could watch for further clues? 

```
    environment:
      - TZ=America/Toronto
      - USER_ID=1004
      - GROUP_ID=1005
    volumes:
      - /hdd12tbpool/docker/crashplan/config:/config:rw
      - /hdd12tbpool/data:/storage:ro
```

**Comment by Juggler00** [comment]
Reading through the current "Issues", I see that `config/log/service.log.0` might hold some further clues. I tailed this file and navigated through the UI to "Manage Files" and selected "storage". When I then clicked on "shared" and was presented with an empty directory in the UI, the log showed the following:
```
[05.22.20 09:25:02.989 INFO  77076841-201 backup42.service.ui.UIController] UI:: UserActionResponse: GetChildrenPathsResultMessage@512722678[  ], parent=V3[cs]-/storage/;w, children=5, totalCount=5 ]
```
Is there anything here to give a clue as to why I'm not seeing any of the folder's content?
